src/main/java/org/opentripplanner/model/plan/ItinerariesCalculateLegTotals.java

```diff
@@ -42,9 +43,14 @@ private void calculate(List<Leg> legs) {
         if (!leg.isInterlinedWithPreviousLeg()) {
           ++nTransitLegs;
         }
-      } else if (leg.isStreetLeg()) {
+      } else if (leg instanceof StreetLeg streetLeg) {
         nonTransitDuration = nonTransitDuration.plus(dt);
```

Comment:
This is really a minor issue, but there seems to be no need for assigning leg to 'streetLeg', as leg supports all required methods. Also, walk duration could use already fetched dt value.

Reply:
You're right, that was a remnant from another version of this code.

It's fixed in 4596d2d78675e9b7ed2ee5556d7c3b64027bea38